src/components/new-request/search.vue

```diff
@@ -639,13 +654,19 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.entity_type_cd = this.business?.legalType || null
     this.setCorpNum(business?.identifier || null)
     this.setEntityTypeCd(this.business?.legalType)
-    if (this.isConversion && this.isBenBusiness) {
-      this.setConversionType(NrRequestTypeCodes.CONVERT_CORP)
-    }
-    // clear all related fields when clearing business search/fetch for alter
-    if (this.isConversion && !this.business) {
-      this.setConversionType(null)
-      this.selectedCompanyType = null
+
+    if (this.isConversion) {
```

Comment:
Add comment explaining what you're doing here.

Reply:
Add comment to code, please :)

Reply:
The new comments aren't as detailed as your comments above (which I think are super useful).